docs/pages/docs/components/button.mdx

```diff
@@ -92,3 +93,142 @@ These sizes are standardized across form components.
 </Container>
 `}
 />
+## types
+
+<Playground
+  code={`
+<Container row direction={['column', 'row', 'row']} gap={10}>
+  <Container align="center" justify={"center"}>
+    <Button type={"primary"}>Primary</Button>
+  </Container>
+
+  <Container align="center" justify={"center"}>
```

Comment:
Not necessary in this use case, remove the `align` and `justify` props, example:
``<Container align="center" justify={"center"}>`` ➡️ ``<Container>``